src/tests/wallet_test.ts

```diff
@@ -1318,4 +1320,59 @@ describe("Wallet Class", () => {
       expect(trades.length).toBe(2);
     });
   });
+
+  describe("#createWebhook", () => {
+    let wallet: Wallet;
+    let addressList: AddressModel[];
+    let walletModel: WalletModel;
+    const existingSeed = "000102030405060708090a0b0c0d0e0f101112131415161718191a1b1c1d1e1f";
+    const { address1, address2, address3, wallet1PrivateKey, wallet2PrivateKey } =
+      generateWalletFromSeed(existingSeed, 3);
+
+    beforeEach(async () => {
+      jest.clearAllMocks();
+      addressList = [
+        {
+          address_id: address1,
+          network_id: Coinbase.networks.BaseSepolia,
+          public_key: wallet1PrivateKey,
+          wallet_id: walletId,
+          index: 0,
+        },
+        {
+          address_id: address2,
+          network_id: Coinbase.networks.BaseSepolia,
+          public_key: wallet2PrivateKey,
+          wallet_id: walletId,
+          index: 1,
+        },
+      ];
+      walletModel = {
+        id: walletId,
+        network_id: Coinbase.networks.BaseSepolia,
+        default_address: addressList[0],
+        feature_set: {} as FeatureSet,
+      };
+      wallet = Wallet.init(walletModel, existingSeed);
+    });
+
+    it("should create a webhook for the default address", async () => {
+      const webhookObject = new Webhook({
```

Comment:
you can use `Webhook.create()` instead of `new` and undo the public constructor change